lesson09/index.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+function createListDom(itemList) {
+    const ulElement = document.getElementById("ul");
```

Comment:
Q: 
Why getting `ul` in Local scope?
getting `ul` is too in createLoadImage Fn .

Reply:
As you say, the intention is to avoid global scope as much as possible.
However, since the same description is repeated, only ul has been modified to be declared globally.

17cd379